openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx

```diff
@@ -211,9 +203,7 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
       displayName: data.name ?? ingestionPipeline.displayName,
       airflowConfig: {
         ...ingestionPipeline?.airflowConfig,
-        scheduleInterval: isEmpty(data.repeatFrequency)
-          ? undefined
-          : data.repeatFrequency,
+        scheduleInterval: isEmpty(data.cron) ? undefined : data.cron,
```

Comment:
In which case we have an empty cron? can we avoid that so we don't need such a check?

Reply:
My bad, we have a check for that input not to be empty. I'll remove the check.

Reply:
It seems you miss to push the changes here?

Reply:
pushed the changes